tsl/src/compression/api.c

```diff
@@ -460,12 +460,20 @@ compress_chunk_impl(Oid hypertable_relid, Oid chunk_relid)
 	compresschunkcxt_init(&cxt, hcache, hypertable_relid, chunk_relid);
 
 	/* acquire locks on src and compress hypertable and src chunk */
+	ereport(LOG,
```

Comment:
Messages about locks might be too verbose for LOG. Maybe DEBUG1? Or INFO under GUC for interactive messages.

Reply:
@akuzm locking is another area which is suspect during chunk_compression. I am inclined to leave it as a LOG for now and if we see any adverse reports about chatty logs then we can take a look again.